lib/driver/mysql.js

```diff
@@ -289,14 +291,33 @@ var MysqlDriver = Base.extend({
     if (params.length > 0 && Array.isArray(params[0])) {
       params = params[0];
     }
-
     return [sql, params, callback];
   },
 
   all: function() {
-    var args = this._makeParamArgs(arguments);
+    var args = this._makeParamArgs(arguments);		
     return this.connection.query.apply(this.connection, args);
   },
+	
+	/**
+	 * Queries the migrations table
+	 *
+	 * @param callback
+	 */
+	_allLoadedMigrations: function(callback) {
+    var sql = 'SELECT * FROM migrations ORDER BY run_on DESC, name DESC';
```

Comment:
There was a commit that allows to define the name of the migrations table. This is still static, needs merge.